kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -1128,90 +1128,28 @@ define([
          *
          * Creating a new bulk import cell returns a Promise, so this returns a Promise.
          */
-        initBulkImport: function () {
+        initImport: function () {
```

Comment:
Simplified `initBulkImport` and `initImportApp` into this function, which just compiles all selected files with their type and passes them off to be imported.